features/editTask.feature

```diff
@@ -0,0 +1,28 @@
+@editTask
+Feature: Editing a task as a teacher and save or discard
+
+    Background: teacher is logged in on the Schul-Cloud homepage
+    
+    Given teacher arrives on the Schul-Cloud homepage
+
+    Scenario Outline: The user logs in as a teacher and edits an existing task
+    When teacher arrives on the Schul-Cloud homepage
+    Given teacher logs in with email <teacherUsername> and password <teacherPassword>
+	Given teacher accepts data protection
+	Then he should click the task-button in the dashboard-sidebar
+    Then he should click the edit-task-button
+    Then he should change the taskname to <taskname> in the name field
+    Then he should change the taskbody to <taskbody> in the taskbody field
+    Then he should change the course to Keine Zuordnung in the dropdown
+    Then he should toggle the allow groupwork checkbox
+    Then he should change the begin and due date
+    Then he should toggle the private task checkbox
+    Then he should toggle the public submissions checkbox
+    Then he should save the changes by clicking on the save-button
+    Then he goes to the tasks page 
+    Then he checks if the new taskname is <taskname>
+    Then he checks if the new taskbody is <taskbody>
+
```

Comment:
have a look here https://docs.hpi-schul-cloud.org/display/CQA/best+practices%3A+feature+files%2C+scenarios%2C+background%2C+scenario+outlines
 and change steps (i mean Then, When and And, there are some hints)